ivy/data_classes/array/linear_algebra.py

```diff
@@ -361,6 +361,60 @@ def inner(
         *,
         out: Optional[ivy.Array] = None,
     ) -> ivy.Array:
+        """
+        Return the inner product of two vectors ``self`` and ``x2``.
+
+        Parameters
+        ----------
+        self
+            first one-dimensional input array of size N.
+            Should have a numeric data type.
+            a(N,) array_like
+            First input vector. Input is flattened if not already 1-dimensional.
+        x2
+            second one-dimensional input array of size M.
+            Should have a numeric data type.
+            b(M,) array_like
+            Second input vector. Input is flattened if not already 1-dimensional.
+        out
+            optional output array, for writing the result to.
+            It must have a shape that the inputs broadcast to.
+
+        Returns
+        -------
+        ret
+            a two-dimensional array containing the inner product and whose
+            shape is (N, M).
+            The returned array must have a data type determined by Type Promotion Rules.
+
+        Examples
+        --------
+        Matrices of identical shapes
+        >>> x = ivy.array([[1., 2.], [3., 4.]])
+        >>> y = ivy.array([[5., 6.], [7., 8.]])
+        >>> d = ivy.inner(x, y)
```

Comment:
These examples should be x.inner(y) not ivy.inner(x, y)

Reply:
Thank you for your review! :) I applied the suggested changes now.